### ISSUE TITLE
Make database

### ISSUE THREAD

**Issue by danman01** [danman01/wise_coders_app#1]
Also seed with interests and code schools

**Comment by danman01** [comment]
ERD:


 users
   - password
   - token
   - email
   - is_admin

user_profiles
   - name
   - wise_coders_slack_handle
   - phone
   - github
   - intro # (short bio) 

user_code_schools
  - user_id
  - code_school_id
  - start_date #when did you start?
  - end_date # when did you end?

code_schools
  - name # general assembly, freeCodeCamp, etc
  - description # what's this school all about

user_interests
  - interest_id
  - user_id
  - experience # how long have you worked with this?
  - expertise # level 1-10 how well you know it
  - summary # describe your interest
  - want_to_teach # any desire in sharing / teaching this interest?
  - want_to_learn # any desire in learning more about this?

interests
  - name # ruby, react, etc.

user_events
  - user_id
  - event_id
  - quantity # how many people are you bringing?
  - working_on # what are you working ?
  - share_tech # interested in sharing any tech topic??
  - teach_tech # interested in teaching any tech topic?

events
  - name
  - start_date
  - end_date
  - duration
  - description

event_happenings
  - event_id
  - happening_id
  - user_id
  - duration
  - start_time
  - end_time
  - notes

happenings
 - user_id # creator
 - interest_id
 - description

event_repeats
  - event_id
  - days_of_week
  - weeks_of_month
  - months_of_year
  - repeat_start_date
  - repeat_end_date
  - number_of_repeats

**Comment by Natesuri** [comment]
I'd love to pair program with someone on this. Still brushing off some Rails cobwebs right now.

**Comment by Aimeelynnramirez** [comment]
Hello there! I’ll be here if so you like me to set up the seeded file

**Comment by danman01** [comment]
> I'd love to pair program with someone on this. Still brushing off some Rails cobwebs right now.

Certainly, if anyone else would like to join in on creating this and pair programming, maybe we can pick a time this week to share a screen over zoom or meet at URI 319 cc @Natesuri

**Comment by Natesuri** [comment]
I'm available on Thursday to pair-program. I'll see you in URI 319 @ 2pm.

**Comment by Natesuri** [comment]
Hey Danny, 

Can a happening only have one associated interest? I'm hoping to set up the happening and event_happenings today. 

Also, how did you imagine the event_repeats working?

Happy to jump on a call to discuss, if that's easier.
